### ISSUE TITLE
loss: nan - val_loss: nan in binary classification

### ISSUE THREAD

**Issue by satadru5** [keras-team/keras#9038]
I am running a binary classification with huge columns(1440) and small data set(~4k)but i am always getting nan  logloss value ..please help..
below my sequential model
----------------------------------------------------
X=train_data
	print(X.shape[1])
	input_dims=X.shape[1]
	act_func = 'relu'
	model = Sequential()
	#input layer
	model.add(Dense(input_dims, input_dim=input_dims))
	model.add(BatchNormalization())
	model.add(Activation('tanh'))
	model.add(Dropout(0.3))
	# hidden layers
	model.add(Dense(input_dims))
	model.add(BatchNormalization())
	model.add(Activation(act_func))
	model.add(Dropout(0.3))

	model.add(Dense(input_dims//2))
	model.add(BatchNormalization())
	model.add(Activation(act_func))
	model.add(Dropout(0.3))
	model.add(Dense(input_dims//4, activation=act_func))

	model.add(Dense(1, activation='sigmoid'))

	# compile this model
	model.compile(loss='binary_crossentropy', optimizer='adam' )# you can add several if needed

	print(model.summary())

**Comment by fchollet** [comment]
- Please format your code properly: https://help.github.com/articles/creating-and-highlighting-code-blocks/
- When are you observing a NaN loss?

**Comment by satadru5** [comment]
from the beginning of the iteration it is giving nan loss.
